packages/orbit-components/src/Select/index.js

```diff
@@ -109,6 +111,7 @@ const StyledSelect = styled(
   width: 100%;
   color: ${({ customValueText }) => customValueText && "transparent !important"};
   transition: box-shadow ${({ theme }) => theme.orbit.durationFast} ease-in-out;
+  color: ${({ customValueText }) => customValueText && "transparent"};
```

Comment:
There is already declaration of the same CSS property & value two lines above. Is it really needed?